File: 2016/gara2/scrigni/att/scrigni.pas
function scosse(N: longint): double;
begin
    (* Mettete qui il codice della soluzione *)
    scosse := 42;
end;


var
    N, i  : longint;
    fr, fw: text;

begin
    assign(fr, 'input.txt');
    assign(fw, 'output.txt');
    reset(fr);
    rewrite(fw);
    readln(fr, N);

    writeln(fw, scosse(N));
    close(fr);
    close(fw);
end.

File: ed2/gara2/scrigni/att/scrigni.c
#include <stdio.h>
#include <assert.h>

double scosse(int N) {
    return 42;
}


int main() {
    FILE *fr, *fw;
    int N;

    fr = fopen("input.txt", "r");
    fw = fopen("output.txt", "w");
    assert(1 == fscanf(fr, "%d", &N));
    fprintf(fw, "%.6f\n", scosse(N));
    fclose(fr);
    fclose(fw);
    return 0;
}
